src/NexusMods.App.UI/Pages/Sorting/LoadOrder/ILoadOrderItemModel.cs

```diff
@@ -13,4 +13,24 @@ public interface ILoadOrderItemModel : ITreeDataGridItemModel<ILoadOrderItemMode
     
     public string ModName { get; }
     public bool IsActive { get; }
+    public string DisplaySortIndex { get; }
+    
+    /// <summary>
+    /// Converts a zero-based index to an ordinal number string with the appropriate suffix.
+    /// </summary>
+    /// <param name="sortIndex">The zero-based index to convert</param>
+    /// <returns>A string representing the ordinal number with the appropriate suffix.</returns>
+    internal static string ConvertZeroIndexToOrdinalNumber(int sortIndex)
```

Comment:
If you put it here I think you can make it protected maybe?
But I think having it in the VM implementation makes sense since it is an implementation detail.
I guess it doesn't matter too much.